dynamax/linear_gaussian_ssm/inference.py

```diff
@@ -13,22 +15,23 @@ class LGSSMParams:
     However, they can also accept a ssm.lgssm.models.LinearGaussianSSM instance,
     if you prefer a more object-oriented approach.
     """
-
     initial_mean: chex.Array
     initial_covariance: chex.Array
     dynamics_matrix: chex.Array
-    dynamics_input_weights: chex.Array
-    dynamics_bias: chex.Array
     dynamics_covariance: chex.Array
     emission_matrix: chex.Array
-    emission_input_weights: chex.Array
-    emission_bias: chex.Array
     emission_covariance: chex.Array
 
+    # Optional parameters (code below assumes zero otherwise)
+    dynamics_input_weights: chex.Array = None
+    dynamics_bias: chex.Array  = None
+    emission_input_weights: chex.Array = None
+    emission_bias: chex.Array = None
+
 
 # Helper functions
 _get_params = lambda x, dim, t: x[t] if x.ndim == dim + 1 else x
-
+_default = lambda x, shape: x if x is not None else jnp.zeros(shape)
```

Comment:
maybe call this `_zeros_if_none` to be consistent with models/lgssm.py

Reply:
Good call, done.